tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -57,21 +57,22 @@ def copy(inp: Field[[IDim], float64]) -> Field[[IDim], float64]:
     assert np.allclose(a_I_float, b_I_float)
 
 
-@pytest.mark.skip(reason="no lowering for returning a tuple of fields exists yet.")
 def test_multicopy(fieldview_backend):
-    a_I_float = np_as_located_field(IDim)(np.random.randn(size).astype("float64"))
-    b_I_float = np_as_located_field(IDim)(np.random.randn(size).astype("float64"))
-    out_I_float = np_as_located_field(IDim)(np.zeros((size,), dtype=float64))
-    out_I_float_1 = np_as_located_field(IDim)(np.zeros((size,), dtype=float64))
+    inp0 = np_as_located_field(IDim)(np.random.randn(size).astype("float64"))
+    inp1 = np_as_located_field(IDim)(np.random.randn(size).astype("float64"))
```

Comment:
```suggestion
    inp1 = np_as_located_field(IDim)(np.random.randn(size).astype("float32"))
```
Just to be sure this also works for different types.

Reply:
done